### PR TITLE
Bugfix/zk UI 201 workflows tab two scrollbars fix

### PR DESCRIPTION
**Pull request by ognyan-kostadinov**

### CODE REVIEW

File: src/react/account/AwsPaginatedResourceTable.tsx [hunk 1]
```diff
@@ -165,6 +165,7 @@ const AwsPaginatedResourceTable = <ENTITY, PREPARED_ENTITY = ENTITY>({
           separationLineVariant="backgroundLevel1"
           backgroundVariant="backgroundLevel3"
           customItemKey={getItemKey}
+          hasScrollbar={false}
```
Comment (ChengYanJin) [inline src/react/account/AwsPaginatedResourceTable.tsx]: can be removed

Comment (ognyan-kostadinov) [reply src/react/account/AwsPaginatedResourceTable.tsx]: Fixed

**Review by ChengYanJin — APPROVED** [review]
LGTM!

**Pull request merged**